api/postcodesearch.go

```diff
@@ -180,6 +181,9 @@ func (api *SearchAPI) getPostcodeSearch(w http.ResponseWriter, r *http.Request)
 func setJSONContentType(w http.ResponseWriter) {
 	w.Header().Set("Content-Type", "application/json")
 }
+func setAccessControl(w http.ResponseWriter) {
```

Comment:
Do we need to set this on the getParentSearch function too?

Reply:
That is a good shout, have added it to that too now